src/test/resources/test.properties

```diff
@@ -2,11 +2,12 @@ mnemonic1=humble poem better wheat work unhappy penalty company steak material c
 rawseed=0xa69e031e4560020684808aa735ea01b20df8b51154e1bfcf9a53ccd0266cc94d
 mnemonic2=chapter chaos smart chat please way horn crouch park audit deputy cupboard
 email=xilos67091@laserlip.com
-phoneApproved =38267873350
-phonePending =123456782273
-phoneFailed =123456782290
-phoneRejected =123456782268
+phoneApproved =67873350
+phonePending =23456782273
+phoneFailed =23456782290
+phoneRejected =23456782268
 testOtp=123456
 referrerLink1 =polkaswap.io/#/referral/cnRY1Xy3Mtbw3xS6XYcm7T4UFGC6k4wzfHCxEydrGZ2ssQS8S
 referrerLink2=polkaswap.io/#/referral/cnUXQmtmidknFFyZFvMdFJuzUY7eB91y9ygEVTzibWZZCmZb9
-assetId=cnUXQmtmidknFFyZFvMdFJuzUY7eB91y9ygEVTzibWZZCmZb9
\ No newline at end of file
+assetId=cnUXQmtmidknFFyZFvMdFJuzUY7eB91y9ygEVTzibWZZCmZb9382
+countryCode=Montenegro
```

Comment:
not sure we should put countries in config. it doesn't depend on environment or a user. maybe just a variable in pageobject will do for now.

Reply:
We have phone numbers of different countries in properties